x-pack/plugins/fleet/server/services/output.ts

```diff
@@ -668,6 +685,11 @@ class OutputService {
       savedObjectType: OUTPUT_SAVED_OBJECT_TYPE,
     });
 
+    await deleteOutputSecrets({
```

Comment:
I wonder if we should change the order, to delete the output first (line 693), and then the secrets? 
Now if the secret deletion succeeds, and the output deletion fails, the state will be inconsistent.

Reply:
Yes you're right 👍